server/server.go

```diff
@@ -118,6 +118,9 @@ type Server struct {
 	stats *statistics
 	// GetCertificate is used for loading certificates.
 	GetCertificate GetCertificate
+	// Sealer is called for Seal and Unseal operations.
+	// Sealer can return a gokeyless.Error to send a custom error code.
+	Sealer func(*gokeyless.Operation) ([]byte, error)
```

Comment:
ah, yeah, didn't mean they behave identically, just that they have the same method signature. Essentially I don't see it as much different to switch inside the function.

But again, weak preference.